clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/ClioIndex.scala

```diff
@@ -21,8 +22,12 @@ trait ClioIndex {
 
   val commandName: String
 
+  val keyFieldNames: Seq[String]
+
   type KeyType <: IndexKey
 
+  val keyTypeTag: TypeTag[KeyType]
```

Comment:
Why do we need this?

Reply:
Its to get the field names of they keys. You mentioned there's a way to get it from the `keyMapper`?

Reply:
`keyMapper.fields.keys` will give you camel-cased versions, which you can then snake-case-ify.